src/databricks/labs/remorph/reconcile/query_builder/hash.py

```diff
@@ -0,0 +1,58 @@
+from io import StringIO
```

Comment:
This filename overlaps with hash() builtin function. Rename to avoid problems

Reply:
Done. Changed the file name to hash_query and test case respectively. I fixed most of the fmt issues. But still giving some weird fmt issues.

Reply:
Made structural changes up on suggestion by Sundar